core/src/main/java/org/polypheny/db/algebra/core/lpg/LpgModify.java

```diff
@@ -53,12 +53,12 @@ protected LpgModify( AlgOptCluster cluster, AlgTraitSet traits, E graph, AlgNode
 
     @Override
     public String algCompareString() {
-        return "$" + getClass().getSimpleName() +
+        return getClass().getSimpleName() +
```

Comment:
Operation is not part of this algCompareString. Intentional?

Reply:
Added.